config/gcos-strict.conf

```diff
@@ -239,6 +239,7 @@ assign-ext-dyn:				unconformable
 assign-disk-from:			unconformable
 vsam-status:				unconformable
 self-call-recursive:			skip
+record-contains-depending-clause:	ok
```

Comment:
The GCOS docs say "The DEPENDING ON clause in format 3 is kept for compatibility with previous releases" - so should this be marked as "obsolete"? This would only raise a warning if explicit requested.

Reply:
Yes indeed. Thanks for pointing this out.